zrml/prediction-markets/src/benchmarks.rs

```diff
@@ -371,35 +352,37 @@ benchmarks! {
     }: { Pallet::<T>::handle_expired_advised_market(&market_id, market)? }
 
     internal_resolve_categorical_reported {
```

Comment:
You benchmark it (verify the benchmark) in another PR?

Reply:
yes.